### PR TITLE
add keymap for meishi2

### PR DESCRIPTION
**Pull request by MakotoKurauchi**
## Description

Keymap to make the best use of LEDs on Pro Micro.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/meishi2/keymaps/led_status/keymap.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+};
+
+void matrix_init_user(void) {
+  setPinOutput(PRO_MICRO_LED_TX);
+  setPinOutput(PRO_MICRO_LED_RX);
+}
+
+void matrix_scan_user(void) {
+
+}
```
Comment (tzarc) [inline keyboards/meishi2/keymaps/led_status/keymap.c:34]:
```suggestion
```

File: keyboards/meishi2/keymaps/led_status/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
```
Comment (tzarc) [inline keyboards/meishi2/keymaps/led_status/config.h:19]: This file should not need to be present.

File: keyboards/meishi2/keymaps/led_status/keymap.c [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    writePinLow(PRO_MICRO_LED_TX);
+  } else {
+    writePinHigh(PRO_MICRO_LED_TX);
+  }
+  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+    writePinLow(PRO_MICRO_LED_RX);
+  } else {
+    writePinHigh(PRO_MICRO_LED_RX);
+  }
+}
```
Comment (zvecr) [inline keyboards/meishi2/keymaps/led_status/keymap.c:47]:
```suggestion
bool led_update_user(led_t led_state) {
    writePin(PRO_MICRO_LED_TX, led_state.num_lock);
    writePin(PRO_MICRO_LED_RX, led_state.caps_lock);

  return true;
}
```

File: keyboards/meishi2/keymaps/led_status/config.h (hunk 2 above)
Comment (MakotoKurauchi) [reply keyboards/meishi2/keymaps/led_status/config.h:19]: thanks

File: keyboards/meishi2/keymaps/led_status/keymap.c (hunk 3 above)
Comment (MakotoKurauchi) [reply keyboards/meishi2/keymaps/led_status/keymap.c]: Thanks for the suggestion.
However, the LED lighting was reversed, so I fixed it.

File: keyboards/meishi2/keymaps/led_status/keymap.c [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define PRO_MICRO_LED_TX D5
+#define PRO_MICRO_LED_RX B0
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT( /* Base */
+    LCTL(KC_Z),  LCTL(KC_X),  LCTL(KC_C), LCTL(KC_V) \
```
Comment (fauxpark) [inline keyboards/meishi2/keymaps/led_status/keymap.c:23]:
```suggestion
    LCTL(KC_Z),  LCTL(KC_X),  LCTL(KC_C), LCTL(KC_V)
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**